python/ray/worker.py

```diff
@@ -636,6 +637,9 @@ def init(
         _metrics_export_port: Optional[int] = None,
         _system_config: Optional[Dict[str, str]] = None,
         _tracing_startup_hook: Optional[Callable] = None,
+        _raylet_socket_name:_raylet_socket_name[str] = None,
+        _plasma_store_socket_name: Optional[str] = None,
```

Comment:
we need to control this because in prod, within our docker containers, the default locations of those might not exist or be writeable